@plotly/webpack-dash-dynamic-import/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Plotly
```

Comment:
FYI, as I wrote in https://github.com/plotly/dash-core/issues/324

> I've seen some major packages like React drop the copyright year - that simplifies managing these and apparently doesn't have a strong downside, so I'd propose we do the same.

Reply:
Will bump to 2021 for now and until the discussion on this happens